src/dashboard/src/settings/common.py

```diff
@@ -340,3 +360,15 @@ def get_env_variable(var_name):
 TEXTAREA_WITH_HELP_ATTRS = {'rows': '4', 'class': 'span11 has_contextual_help'}
 INPUT_ATTRS = {'class': 'span11'}
 INPUT_WITH_HELP_ATTRS = {'class': 'span11 has_contextual_help'}
+
+SHIBBOLETH_REMOTE_USER_HEADER = 'HTTP_EPPN'
+SHIBBOLETH_ATTRIBUTE_MAP = {
```

Comment:
These are other values that Archivematica takes from the Shibboleth info. Ones that match user fields will be applied to the user object. Others are handled manually.